backup_and_restore/application_backup_and_restore/installing/about-oadp-1-3-data-mover.adoc

```diff
@@ -0,0 +1,57 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-oadp-1-3-data-mover"]
+= About the OADP 1.3 Data Mover
+include::_attributes/common-attributes.adoc[]
+:context: about-oadp-1-3-data-mover
+
+toc::[]
+
+OADP 1.3 includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses Kopia as the uploader mechanism to read the snapshot data and write to the unified repository.
+
+OADP supports CSI snapshots on the following:
+
+* Red Hat OpenShift Data Foundation
+* Any other cloud storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
+
+:FeatureName: The OADP built-in Data Mover
+include::snippets/technology-preview.adoc[]
+
+[id="enabling-oadp-1-3-data-mover"]
+== Enabling the built-in Data Mover
+
+To enable the built-in Data Mover, you must include the CSI plugin and enable the node agent in the `DataProtectionApplication` custom resource (CR). The node agent is a Kubernetes daemonset that hosts data movement modules. These include the Data Mover controller, uploader, and the repository. 
+
+.Example `DataProtectionApplication` manifest
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: dpa-sample
+spec:
+  configuration:
+    nodeAgent:
+      enable: true <1>
+      uploaderType: kopia
```

Comment:
uploaderType can be restic or kopia. Velero will always use kopia irrespective of uploaderType set on DPA CR.